### PR TITLE
DevWS submission for a number of Sengled bulbs

### PR DESCRIPTION
**Pull request by sxl1987**

### CODE REVIEW

**Comment by shrakuma** [comment]
@MAblewiczS can you please triage this to your team for review 

cc @greens @tpmanley

**Comment by MAblewicz** [comment]
@sxl1987  Please bring you fork up to date to SmartThingsCommunity:master. You are removing FP for other partners.

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 1]
```diff
@@ -58,9 +58,13 @@ metadata {
...
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-N13A", deviceJoinName: "Sengled Extra Bright Soft White"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E11-N14A", deviceJoinName: "Sengled Extra Bright Daylight"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E21-N13A", deviceJoinName: "Sengled Soft White"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0702, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E21-N14A", deviceJoinName: "Sengled Daylight"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FEDC", outClusters: "000A, 0019", manufacturer: "LDS", model: "ZBT-DIMLight-GLS0006", deviceJoinName: "Smart Bulb"
```
Comment (MAblewicz) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: Please use tabs as indentation and keep correct file formating.

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 2]
```diff
@@ -69,12 +73,6 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0301, FC01", manufacturer: "ubisys", model: "D1 (5503)", deviceJoinName: "INGENIUM ZB Universal Dimming Module ZBM01d"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", manufacturer: "Megaman", model: "AD-DimmableLight3001", deviceJoinName: "INGENIUM ZB LED Classic"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019", deviceId: "0101", manufacturer: "AduroSmart Eria", model: "AD-DimmableLight3001", deviceJoinName: "Eria ZigBee Dimmable Bulb"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FC82", outClusters: "0019", manufacturer: "innr", model: "RF 263", deviceJoinName: "INNR Smart Filament Bulb Vintage"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FC82", outClusters: "0019", manufacturer: "innr", model: "BF 263", deviceJoinName: "INNR Smart Filament Bulb Vintage"
```
Comment (MAblewicz) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: @sxl1987 Please bring you fork up to date to SmartThingsCommunity:master. You are removing FP for other partners.

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy [hunk 3]
```diff
@@ -66,8 +65,8 @@ metadata {
 		// Leedarson/Ozom
 		fingerprint profileId: "0104", inClusters: "0000, 0004, 0003, 0006, 0008, 0005, 0300", outClusters: "0019", manufacturer: "LEEDARSON LIGHTING", model: "5ZB-A806ST-Q1G", deviceJoinName: "Ozom Multicolor Smart Light"
 		// Sengled
-		fingerprint profileId: "0104", inClusters: "0000,0003,0004,0005,0006,0008,0300,0702,0B05,FC03,FC04", outClusters: "0019", manufacturer: "sengled", model: "E11-N1EA", deviceJoinName: "Sengled Element Color Plus"
```
Comment (MAblewicz) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy:77]: @sxl1987 Are you sure about removing/changing those Fingerprint. It looks like those are diffrent models?

**Comment by MAblewicz** [comment]
@sxl1987 Also please resolve merge conficts (maybe some of them are related to outdated fork).

**Comment by tpmanley** [comment]
FYI I just checked in a change that organized all the fingerprints by manufacturer but this is causing some conflicts in your PR. It might be easiest to just reset your branch to the latest `upstream/master`, re-add your changes to the appropriate section and then force push. Something like this:

```
git fetch
git reset --hard upstream/master
<manually re-add your fingerprints to the correct section>
git push -f origin devws_1
```

**Comment by shrakuma** [comment]
@tpmanley @MAblewiczS @greens  can you please check if the latest updates made are ok?

File: devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy [hunk 4]
```diff
@@ -22,59 +22,87 @@ metadata {
...
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FC82", outClusters: "0019", manufacturer: "innr", model: "BF 265", deviceJoinName: "Innr Smart Filament Bulb"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0019", manufacturer: "innr", model: "AE 260", deviceJoinName: "Innr Smart Bulb"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0019", manufacturer: "innr", model: "BE 220", deviceJoinName: "Innr Smart Flood Light White"
-		fingerprint mfr:"innr", model:"RF 265", deviceJoinName: "Innr Smart Filament Bulb White" // raw description: 01 0104 0101 01 09 0000 0003 0004 0005 0006 0008 0B05 1000 FC82 01 0019
```
Comment (MAblewicz) [inline devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy]: @sxl1987 Why is this fingerprint removed (fingerprint mfr:"innr", model:"RF 265", deviceJoinName: "Innr Smart Filament Bulb White") . You did some cleaning and for rest removed FP you re-add them but not this one.

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy [hunk 5]
```diff
@@ -63,11 +67,13 @@ metadata {
...
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, FC01", outClusters: "0019", manufacturer: "LEDVANCE", model: "Outdoor Pathway RGBW", deviceJoinName: "SYLVANIA Outdoor Pathway Full Color"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0B05, FC01", outClusters: "0019", manufacturer: "LEDVANCE", model: "Flex RGBW Pro", deviceJoinName: "SYLVANIA Smart Flex 11 RGBW"
+
 		// Leedarson/Ozom
 		fingerprint profileId: "0104", inClusters: "0000, 0004, 0003, 0006, 0008, 0005, 0300", outClusters: "0019", manufacturer: "LEEDARSON LIGHTING", model: "5ZB-A806ST-Q1G", deviceJoinName: "Ozom Multicolor Smart Light"
+
 		// Sengled
-		fingerprint profileId: "0104", inClusters: "0000,0003,0004,0005,0006,0008,0300,0702,0B05,FC03,FC04", outClusters: "0019", manufacturer: "sengled", model: "E11-N1EA", deviceJoinName: "Sengled Element Color Plus"
-		fingerprint profileId: "0104", inClusters: "0000,0003,0004,0005,0006,0008,0300,0702,0B05,FC03,FC04", outClusters: "0019", manufacturer: "sengled", model: "E12-N1E", deviceJoinName: "Sengled Element Color Plus"
```
Comment (MAblewicz) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy:78]: @sxl1987 Are you sure about removing/changing those Fingerprint. It looks like those are diffrent models?

**Review by MAblewicz — CHANGES_REQUESTED** [review]
@sxl1987 Please check my comments and resolve them.
@tpmanley  Is it ok to do this kind of cleaning in DTH by partner. It was very hard to quickly review this.

**Review by tpmanley — APPROVED** [review]

**Review by MAblewicz — APPROVED** [review]
Look ok now.

**Pull request merged**